app/views/Layout.scala.html

```diff
@@ -54,6 +54,7 @@
     }
 
     @additionalHead = {
+    <script @{CSPNonce.attr} src='@controllers.routes.Assets.versioned("javascripts/main.js")'></script>
```

Comment:
This section may need indenting?

@additionalHead = {
    <script...
    
Rather than

@additionalHead = {
<script...